Content.Server/_Goobstation/GameTicking/Rules/HereticRuleSystem.cs

```diff
@@ -32,6 +31,8 @@ public sealed partial class HereticRuleSystem : GameRuleSystem<HereticRuleCompon
     [Dependency] private readonly HellWorldSystem _hell = default!;
 
     public readonly SoundSpecifier BriefingSound = new SoundPathSpecifier("/Audio/_Goobstation/Heretic/Ambience/Antag/Heretic/heretic_gain.ogg");
+    public readonly SoundSpecifier RiftSpawnSound = new SoundPathSpecifier("/Audio/_Goobstation/Heretic/Ambience/Antag/Heretic/heretic_gain.ogg");
```

Comment:
i hate goob putting these kinds of things in here instead of `HereticRuleComponent` but we fucking ball i guess